src/auth/computeclient.ts

```diff
@@ -33,7 +33,7 @@ export interface ComputeOptions extends RefreshOptions {
 }
 
 export class Compute extends OAuth2Client {
-  private serviceAccountEmail: string;
```

Comment:
nit: readonly maybe? Looks like the java implementation has a getter but no setter